.github/CODEOWNERS

```diff
@@ -562,6 +562,8 @@
 /scripts/run_lighthouse_tests.py @jameesjohn @vojtechjelinek
 /webpack.*.ts @nishantwrp @vojtechjelinek
 
+# Security
+/scripts/linters/warranted_angular_security_bypasses.py @seanlip
```

Comment:
@seanlip, Should we move this to the `# Critical files.` section?

Reply:
That sounds fine to me, since we already have it. @srijanreddy98 is this ok with you?

Reply:
I have changed it to critical files. Although, I would say that critical files are those files, without which our application won't boot-up (something like sys32 in windows). This file doesn't exactly fall into that category. But anyway, I have changed it.